tools/reactive-controllers/element-resolution.md

```diff
@@ -1,6 +1,26 @@
-## Description
+## Overview
 
-An `ElementResolutionController` keeps an active reference to another element in the same DOM tree. Supply the controller with a selector to query and it will manage observing the DOM tree to ensure that the reference it holds is always the first matched element or `null`.
+The `ElementResolutionController` is a [reactive controller](https://lit.dev/docs/composition/controllers/) that maintains an active reference to another element in the same DOM tree. It automatically observes the DOM tree for changes and ensures that the reference it holds is always up-to-date with the first matched element or `null` if no match is found.
+
+### Features
+
+- **Automatic element tracking**: Maintains a live reference to elements matching a CSS selector
+- **DOM observation**: Uses `MutationObserver` to track changes in the DOM tree
+- **Efficient ID resolution**: Optimized path for ID-based selectors
+- **Reactive updates**: Automatically triggers host updates when the resolved element changes
+- **Scope awareness**: Works within Shadow DOM and regular DOM contexts
+
+<!-- ### Usage
+
+```typescript
+import { elementResolverUpdatedSymbol } from '@spectrum-web-components/reactive-controllers/src/ElementResolution.js';
+
+protected override willUpdate(changes: PropertyValues): void {
+    if (changes.has(elementResolverUpdatedSymbol)) {
+        // React to element resolution changes
+    }
+}
+``` -->
```

Comment:
Does this still need to be in here?

Reply:
I will clean these up in a separate commit. Thanks